src/main/kotlin/org/rust/lang/core/completion/RsAttributeCompletionProvider.kt

```diff
@@ -82,10 +83,12 @@ object RsAttributeCompletionProvider : RsCompletionProvider() {
     }
 
     override val elementPattern: ElementPattern<PsiElement>
-        get() {
-            return psiElement(RsElementTypes.IDENTIFIER)
-                .withParent(psiElement<RsPath>().withParent(rootMetaItem))
-        }
+        get() = psiElement(RsElementTypes.IDENTIFIER)
+            .withParent(
+                psiElement<RsPath>()
+                    .with("Unqualified") { it: RsPath -> it.path == null }
```

Comment:
Let's also cover `::foo` case
```suggestion
                    .with("Unqualified") { it: RsPath -> it.coloncolon == null }
```